File: Problem 1/MultiplesOf3And5.js
// Finding Multiples of 3 and 5
function findMultis (sum) {
  // Defining the iterator "i" as specifically a integer
  let i = 0
  // using "i" to loop to 1000, iterating by 1 each time
  for (i = 0; i < 1000; i++) {
    // If "i" modulo 3 leaves a remainder of 0,
    // it is evenly divisible
    if ((i % 3) === 0) {
      // we add the value of "i" to the total sum and
      // continue out of the IF statement
      sum += i
      continue
    // Else, we see if "i" is evenly divisble by 5, again,
    // using the modulo function
    } else if ((i % 5) === 0) {
      // If it is, we add the value of "i" to the total
      // sum number
      sum += i
      continue
    }
  }
  // Once the FOR loop is completed, we print the value of "sum"
  // to the console
  console.log(sum.toString())
}

function main () {
  // Creating the "sum" variable and forcing it to be an integer
  let sum = 0
  // Calling "findMultis", and parsing the value of "sum" to it
  findMultis(sum)
}

main()

File: Problem 1/MultiplesOf3and5.cpp
#include <iostream>
using namespace std;

// Main Function for finding multiples of 3 and 5
void findMultiples(int sum) {
  // For loop that will iterate over x, adding 1 each loop until
  // it equals 1000
  for (int x = 0; x < 1000; x++) {
    // Checking if x is evenly divisible by 3
    if ((x % 3) == 0) {
      // If it is, then we add it to the sum
      sum += x;
      // Next we check if x is evenly divisible by 5
    } else if ((x % 5) == 0) {
      // If it is, we add it to the sum
      sum += x;
    }
  }

  cout << sum;
}


//Main loop for calling the "Find Multiples" function
int main() {
  // Creating the sum variable that will be used to store our
  // result
  int sum = 0;
  // Calling the findMultiples function and parsing the sum variable
  findMultiples(sum);
  return 0;
}
